app/models/uphold_report.rb

```diff
@@ -0,0 +1,3 @@
+class UpholdReport < ApplicationRecord
```

Comment:
Nitpicking, but internally the convention that would be used is `UpholdStatusReport`

Reply:
https://github.com/brave-intl/publishers/pull/2219/commits/c35cb3d2f276a0c1b81ee111a38d9f4b56e8744e